pkg/util/tmplvar/parse.go

```diff
@@ -53,3 +56,61 @@ func parseTemplateVar(v []byte) (name, key []byte) {
 	}
 	return name, key
 }
+
+var tmplVarEnvRegex = regexp.MustCompile(`%%env_.+?%%`)
+
+// ParseTemplateEnvString replaces all %%env_VARIABLE%% from environment
+// variables. When there is an error, use the original string
+func ParseTemplateEnvString(input string) string {
```

Comment:
Yeah, but it is causing circular dependency because autodiscovery calls tagger here:
https://github.com/DataDog/datadog-agent/blob/3eac8994dc782878245732722f1ce8cc7ecbeb3c/pkg/autodiscovery/listeners/service.go
So I have to implement a simplified one.

Reply:
Can we extract this function in another package to break the circular deps?